asset/css/style.css

```diff
@@ -0,0 +1,99 @@
+* {
+  font-family: 'Poppins', sans-serif;
+}
+
+/* Mobile View */
+@media all and (max-width: 375px) {
+  * {
+    padding-left: 5px;
+    padding-right: 5px;
+  }
+
+  #main-section {
+    background-image: url(../images/background.png);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: 100% 95%;
+    height: 588px;
+  }
```

Comment:
Your background image placement is not quite right. Please take a closer look at the Figma design and fix it.

<img width="304" alt="Screen Shot 2022-06-29 at 18 18 26" src="https://user-images.githubusercontent.com/82426682/176561524-b16e6bc6-ed4c-43c5-ad95-004cfafcfdc1.png">

Additionally, you're missing the `border-radius` in the lower-left corner of the section, please add it.

Reply:
Fixed